ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.html

```diff
@@ -182,50 +235,89 @@ <h4>There are no kit requests</h4>
-              <ng-container *ngIf="kitRequest.easypostTrackingReturnUrl != null"><a href="{{kitRequest.easypostTrackingReturnUrl}}" target="_blank">{{kitRequest.trackingReturnId}}</a>
+          <td *ngIf="shippingPage === SENT" [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP' && shippingPage === OVERVIEW}">
+            <!--            tracking return number-->
+            <ng-container *ngIf="kitRequest.easypostTrackingReturnUrl != null"><a href="{{kitRequest.easypostTrackingReturnUrl}}" target="_blank">{{kitRequest.trackingReturnId}}</a>
             </ng-container>
             <ng-container *ngIf="kitRequest.easypostTrackingReturnUrl == null">{{kitRequest.trackingReturnId}}</ng-container>
           </td>
-          <td *ngIf="shippingPage === SENT || shippingPage === OVERVIEW">
+          <td *ngIf="shippingPage === SENT || shippingPage === OVERVIEW" [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP' && shippingPage === OVERVIEW}">
+            <!--            sent date-->
             <ng-container *ngIf="kitRequest.scanDate === 0"> -</ng-container>
             <ng-container *ngIf="kitRequest.scanDate !== 0">{{kitRequest.scanDate | date:'medium'}}</ng-container>
           </td>
-          <td *ngIf="shippingPage === RECEIVED || shippingPage === OVERVIEW">
+          <td *ngIf="shippingPage === RECEIVED || shippingPage === OVERVIEW" [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP'}">
+            <!--            received date-->
             <ng-container *ngIf="kitRequest.receiveDate === 0"> -</ng-container>
             <ng-container *ngIf="kitRequest.receiveDate !== 0">{{kitRequest.receiveDate | date:'medium'}}</ng-container>
           </td>
-          <td *ngIf="shippingPage === SENT || shippingPage === RECEIVED || shippingPage === OVERVIEW">
+          <td *ngIf="shippingPage === SENT || shippingPage === RECEIVED || shippingPage === OVERVIEW" [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP'}">
+            <!--            mf barcode-->
             {{kitRequest.kitLabel}}
           </td>
-          <td>{{kitRequest.realm}}</td>
-          <td>{{kitRequest.kitTypeName}}</td>
+          <td [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP'}">
+            <!--            study-->
+            {{kitRequest.realm}}
+          </td>
+          <td [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP'}">
+            <!--            kit type-->
+            {{kitRequest.kitTypeName}}
+          </td>
+          <td *ngIf="isClinicalStudy && (shippingPage === RECEIVED || shippingPage === OVERVIEW)" [ngClass]="{'highlight_sample': kitRequest.receivedBy === 'BSP'}">
+            <!--            sample type-->
+            <ng-container *ngIf="kitRequest.receivedBy === 'MERCURY'">Clinical Sample</ng-container>
+            <ng-container *ngIf="kitRequest.receivedBy === 'BSP'">Research Sample</ng-container>
+          </td>
```

Comment:
Based on the ticket, shouldn't we also check that  if `kit.sequencingRestriction` is marked as research as well?

Reply:
that would be on the changes for DDP-8569. This ticket is only for adding the highlighting there